drivers/memc/Kconfig.mcux

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 NXP
+# Copyright (c) 2021 Basalte bv
+# SPDX-License-Identifier: Apache-2.0
+
+if HAS_MCUX_FLEXSPI
+
+config MEMC_MCUX_FLEXSPI_HYPERRAM
+	bool "MCUX FlexSPI HyperRAM driver"
+	select MEMC_MCUX_FLEXSPI
```

Comment:
nit: these lines should be added in commit `drivers: memc: Introduce i.MX RT FlexSPI HyperRAM driver`

Reply:
Thanks for the review, this got lost in the rebase :)